roles/pulp/tasks/install.yml

```diff
@@ -210,11 +210,13 @@
         LC_ALL: en_US.UTF-8
         LANG: en_US.UTF-8
       changed_when: false
+      when: pulp_source_dir is undefined
 
     - name: Fail when pulpcore and plugin versions are not compatible.
       fail:
       when:
-        - '"Could not find a version that matches" in compatibility.stderr'
+        - pulp_source_dir is undefined
+        - '"Could not find a version" in compatibility.stderr'
```

Comment:
Are there any instances where the rc would be non-0 that we want it to succeed?

Reply:
yes, when pulp_rpm is being installed and it's version compatibility is fine. pip-compile tries load the setup.py and fails due to missing deps. However, this occurs after the version comparison between it and pulpcore is already finished.

Reply:
OK, let's add a comment about that.

Reply:
I was able to remove the ignore_errors: true after i added the failed_when condition. I'll add a comment anyway though.

Reply:
Awesome, thank you :)